package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "vue",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.6.5",
+    "vue": "^2.6.11",
```

Comment:
Why not Vue 3?

Reply:
Buefy plugin that we are planning to use in this project does not support Vue 3 currently, so we sticked with Vue 2.

Reply:
So maybe it's a good moment to abandon this Buefy thingo. ;)

Reply:
After some investigation, Buefy is nowhere near to be migrated to Vue 3. It seems to be a good idea to drop this package for now.